lambda_functions/README.md

```diff
@@ -24,18 +29,27 @@ Run the following commands to update the alerter.zip (starting in the lambda_fun
 
 ```
     mkdir packages
-    pip install --target ./packages requests
-    pip install --target ./packages <any packages you need>
+    cd packages
+    pip install requests
+    pip install <any packages you need>
     zip -r ../alerter.zip .
     cd ..
     zip alerter.zip alerter.py
 ```
 
 This will allow your lambda to use package dependencies.
 
-## Uploading your lambdas
+## Updating your lambdas
```

Comment:
If we are saying that we always need to manually update the lambdas in the AWS console whenever changes are made to the lambda code then this isn't the right way to go. The deployment pipeline has to be able to build the zip file, push to S3 (or wherever) and terraform then applies the latest version of that file.

Reply:
we could try lambda layers as an alternative

Reply:
Removed file as this is now part of the pipeline